aplib.net-demo/Assets/Scripts/InputManager.cs

```diff
@@ -5,12 +5,14 @@ public class InputManager : MonoBehaviour
 {
     [SerializeField] private MouseLock _mouseLock;
     [SerializeField] private Transform _playerTransform;
+    [SerializeField] private Inventory _inventory;
     private ResetRigidbody _playerRespawn;
     private Movement _playerMovement;
     // TODO: Change when inventory is added
     // Doing it this way for now, change when inventory is implemented.
```

Comment:
Isn't that now?

Reply:
Yea, maar dat is APLIB-98 om de weapons over te zetten naar de inventory system